src/main/java/controller/Calculator.java

```diff
@@ -0,0 +1,23 @@
+package controller;
+
+import service.CalculatorService;
+import view.Input;
+import view.Output;
+
+public class Calculator {
+
+    public void start() {
+        Input input = new Input();
+        String inputValue = input.inputValue();
+
+        // 로직
+
+        CalculatorService calculatorService = new CalculatorService();
+        Integer calculatedValue = calculatorService.calculate(inputValue);
```

Comment:
```suggestion
        int calculatedValue = calculatorService.calculate(inputValue);
```
래퍼클래스일 필요가 있을까요?

null을 다루지 않는다면 primitive 타입을 사용하는 것이 적절할 것 같습니다.

Reply:
넵, 해당 코드 작성할때 null도 리턴할 수 있다는 부분을 고려해서 로직을 만들려고 했는데

Reply:
아, 궁금한게 있는데 예전에 듣기론 반환타입을 primitive타입보단 객체 타입을 사용하는게 좋다고 들었는데 
그게 맞는 말인지 좀 궁금합니다.
( 그 이유는 null을 반환할수도 있기 때문이라고 들었습니다.)

Reply:
보너스를 주제로해서 정말 간단한 상황으로 예시를 들어볼게요

### 보너스가 int 타입일 때 
보너스를 줬지만 알고리즘으로 인해 0원이 지급될 수 있습니다.
이러면 보너스를 안 받았는지, 받았는데 0원인지 헷갈릴 수 있습니다.

### 보너스가 Integer 일 때
보너스를 받지 않으면 null, 받았다면 실제 값이 할당됩니다.

실제 로직은 이렇지 않지만... 이렇게 null 인 상황이 필요하냐 안하냐로 나뉘기 때문에 
맞고 틀리고는 없고 상황에 따라 다를 것 같습니다 😄

추가로 제네릭을 다룰 때 primitive 타입을 사용할 수 없기 때문에 래퍼 클래스를 사용합니다